File: 1st/Hyeseung/Programmers/다음에 올 숫자.java
class Solution {
    public int solution(int[] common) {
        int answer = 0;
        if(common[1] - common[0] == common[2] - common[1])
            answer = common[common.length - 1] + (common[2] - common[1]);
        else
            answer = common[common.length - 1] * (common[2] / common[1]);
        return answer;
    }
}

File: 2st/Yeonwoo/다음에 올 숫자.py
def solution(common):
    answer = 0
    a = common[0]
    d = 0
    r = 0
    if common[1]-common[0] == common[2]-common[1]:
        d = common[1]-common[0]
        answer = common[-1]+d
    else:
        r = common[1]/common[0]
        answer = common[-1]*r
    return answer